From d063ea36295e2bc7a07b83a60eab16f0b7cc7ef8 Mon Sep 17 00:00:00 2001
From: Yusuke Goda <yusuke.goda.sx@renesas.com>
Date: Thu, 21 Dec 2017 13:41:56 +0900
Subject: [PATCH 675/909] arm64: dts: r8a7795-es1-h3ulcb: Add VSPM I/F driver
 node

commit a1a30418b05d2f207dc1e16e1c9a2164a528b864 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Add VSP Manager I/F driver node for H3ULCB board on R8A7795 ES1.x SoC.

Signed-off-by: Yusuke Goda <yusuke.goda.sx@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/renesas/r8a7795-es1-h3ulcb.dts   | 24 +++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
index e259922c6f66..119f06c87ac0 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
@@ -70,8 +70,32 @@
 	mmngrbuf {
 		compatible = "renesas,mmngrbuf";
 	};
+
+	vspm_if {
+		compatible = "renesas,vspm_if";
+	};
 };
 
 &sdhi2 {
 	/delete-property/ mmc-hs400-1_8v;
 };
+
+&vspbc {
+	status = "okay";
+};
+
+&vspbd {
+	status = "okay";
+};
+
+&vspi0 {
+	status = "okay";
+};
+
+&vspi1 {
+	status = "okay";
+};
+
+&vspi2 {
+	status = "okay";
+};
-- 
2.17.1

